controllers/productsController.js

```diff
@@ -72,10 +76,98 @@ module.exports.doCreate = (request, response) => {
             product.save().then(() => {
                 console.log('success')
                 return response.render('registration-success');
```

Comment:
After Product Creation, we should redirect the User to the Product that he/she just created.

should be
```
response.redirect('/product/' + product._id);
```
in this case, he/she gets redirected to the `Product Editor Page` of the newly created product.